internal/cmd/plan.go

```diff
@@ -315,6 +330,44 @@ func PaymentMethodHelper(client *turso.Client, selected string) (bool, error) {
 	}
 }
 
+func PaymentMethodHelperOverages(client *turso.Client) (bool, error) {
```

Comment:
We have 3 almost identical functions like this one. Anything we can do to make this logic more concise?